documentation/man/features/build.rst

```diff
@@ -0,0 +1,38 @@
+========
+kw-build
+========
+
+.. _build-doc:
+
+----------------
+build the kernel
+----------------
+
+SYNOPSIS
+========
+*kw* (*b* | *build*) [-i | \--info] [-n | \--menu] [-d | \--doc]
+
+DESCRIPTION
+===========
+If users invoke this option without parameters, kw will look at the local
+Makefile and, based on that, start to build the project. This option tries to
+take advantage of your hardware by using the **-j** option with the appropriate
+parameter.
+
+OPTIONS
+=======
+-i, \--info:
+    The info option, makes so the build information such as the kernel
+    release name and the total number of modules compiled will be displayed.
+
+-n, \--menu:
+    The menu option invokes the kernel menuconfig. Notice that the default menu
+    config can be changed in the `kworkflow.config` file by setting a different
+    option in *menu_config*. If the user is working in a *cross-compile*
+    environment, it is recommended to use this option to avoid messing with the
+    config file manually.
+
+-d, \--doc:
+    The doc option provides a mechanism for build kernel-doc; by default, it
+    will build htmldocs. Users can change the default documentation output by
+    changing the parameter *doc_type* in the `kworkflow.config` file.
```

Comment:
In the kw.rst, we have a few examples of this feature. Please, add an example section here.